pkgs/development/python-modules/dulwich.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchurl
+, pkgs, gevent, geventhttpclient, mock, fastimport }:
```

Comment:
How can I avoid shadowing of packages from `pkgs` by python packages with the same name in this case?